packages/react-widgets/src/NumberPicker.tsx

```diff
@@ -247,26 +333,26 @@ function NumberPicker(uncontrolledProps) {
     inputRef.current?.focus()
   }
 
-  function increment(event) {
-    return step(pStep, event)
+  function increment(event: React.KeyboardEvent<HTMLDivElement> | React.MouseEvent<HTMLButtonElement>) {
+    return step(pStep!, event)
   }
 
-  function decrement(event) {
-    return step(-pStep, event)
+  function decrement(event: React.KeyboardEvent<HTMLDivElement> | React.MouseEvent<HTMLButtonElement>) {
+    return step(-pStep!, event)
   }
 
-  function step(amount, event) {
+  function step(amount : number, event: React.KeyboardEvent<HTMLDivElement> | React.MouseEvent<HTMLButtonElement>) {
     const nextValue = (value || 0) + amount
 
     handleChange(
-      precision != null ? round(nextValue, precision) : nextValue,
+      precision != null ? round(nextValue, precision) as any as number /*HACK*/ : nextValue,
```

Comment:
oooo this is just a bug

Reply:
I added parseFloat, not sure if this is a fix